File: orchestrator-api/src/main/java/com/example/gwp/orchestrator/OrchestratorApplication.java
package com.example.gwp.orchestrator;

import com.example.gwp.orchestrator.config.properties.GwpProperties;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication
@ConfigurationPropertiesScan(basePackageClasses = GwpProperties.class)
public class OrchestratorApplication {

    public static void main(String[] args) {
        SpringApplication.run(OrchestratorApplication.class, args);
    }
}

File: orchestrator-api/src/main/kotlin/com/example/gwp/orchestrator/OrchestratorApplication.kt
package com.example.gwp.orchestrator

import com.example.gwp.orchestrator.config.properties.GwpProperties
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan

@SpringBootApplication
@ConfigurationPropertiesScan(basePackageClasses = [GwpProperties::class])
class OrchestratorApplication

fun main(args: Array<String>) {
    SpringApplication.run(OrchestratorApplication::class.java, *args)
}
